requirements.txt

```diff
@@ -0,0 +1,11 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+datasets==2.16.1
+evaluate==0.4.1
+pandas==2.1.4
+pydantic==2.5.3
```

Comment:
Where do we use this dependency?

Reply:
it is necessary for implementation of the class Query in service.py

Reply:
Ok😑